src/xdp/trn_transit_xdp.c

```diff
@@ -792,6 +793,22 @@ int _transit(struct xdp_md *ctx)
 
 	int action = trn_process_eth(&pkt);
 
+	/* 
+	 * Look up the entry in the metrics table, the key used is "0" 
```

Comment:
"kind" as a testing platform is tricky. The kind nodes will share the memory on the host machine, which means the eBPF map belong to each kind node co-locates on the same memory. So it would not be feasible to have the eBPF maps of different kind nodes use the same key. We cannot use the same key to differentiate the eBPF maps of different kind nodes.

Instead, I'll use the host ip as the unique key of each kind node, as each kind node has its unique IP in most case